packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -52,5 +52,7 @@ export const messages = {
     'No code coverage information was found for this file. Set "salesforcedx-vscode-core.retrieve-test-code-coverage": true in your user or workspace settings. Then, run Apex tests that include methods in this file. You can run tests from the Apex Tests sidebar or using the Run Tests or Run All Tests code lens within the file.',
   colorizer_statusbar_hover_text: 'Highlight Apex Code Coverage',
   force_sobjects_no_refresh_if_already_active_error_text:
-    'A refresh of your sObject definitions is already underway. If you need to restart the process, cancel the running task.'
+    'A refresh of your sObject definitions is already underway. If you need to restart the process, cancel the running task.',
+  apex_language_server_failed_activate:
```

Comment:
@ruthemmanuelle here's a new generic error message that will be displayed to the user when the error is not specific to Java version misconfiguration. Let me know what you think.

Reply:
Can we give any advice on what to do next?

Reply:
Not really, this message is shown when something goes bad internally in Apex language server but I haven't been able to set a project in a way in which I end up showing this message.

Reply:
Do we want to advise them to file an issue on GitHub? Seems like we'd be curious to know how they got in this state. Maybe say something like, `The Apex Language Server failed to activate. Please file an issue on the salesforcedx-vscode GitHub repository.`?